sycl/doc/extensions/GroupMask/SYCL_INTEL_group_mask.asciidoc

```diff
@@ -146,6 +146,48 @@ The mask is defined such that the least significant bit (LSB) corresponds to the
 |+void flip(id<1> id)+
 |Toggle the value of the bit corresponding to the specified _id_.
 
+|+bool operator==(group_mask rhs) const+
+|Return true if each bit in this mask is equal to the corresponding bit in +rhs+.
+
+|+bool operator!=(group_mask rhs) const+
+|Return true if any bit in this mask is not equal to the corresponding bit in +rhs+.
+
+|+group_mask operator &=(group_mask rhs)+
+|Set the bits of this mask to the result of performing a bitwise AND with this mask and +rhs+.
+
+|+group_mask operator |=(group_mask rhs)+
+|Set the bits of this mask to the result of performing a bitwise OR with this mask and +rhs+.
+
+|+group_mask operator ^=(group_mask rhs)+
+|Set the bits of this mask to the result of performing a bitwise XOR with this mask and +rhs+.
+
+|+group_mask operator <<=(size_t shift)+
+|Set the bits of this mask to the result of shifting its bits _shift_ positions to the left.
+
+|+group_mask operator >>=(size_t shift)+
+|Set the bits of this mask to the result of shifting its bits _shift_ positions to the right.
+
```

Comment:
I think it worth clarifying whether bits which are shifted out are discarded or cycle-shift is performed. Also, what is the default value of "new" bits?

I mean:
```c++
mask m = /* all ones */;
m >>= 1;
if (m[0]) { // is it true or false?
  // ...
}
```

Reply:
Agreed.  Fixed in 11a73fb.